Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAccountProtectionPolicy/MSFT_IntuneAccountProtectionPolicy.psm1

```diff
@@ -0,0 +1,905 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Identity,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.String]
+        $Description,
+
+        [Parameter()]
+        [System.Boolean]
+        $WindowsHelloForBusinessBlocked,
+
+        [Parameter()]
+        [ValidateScript({ ($_ -ge 4) -and ($_ -le 127) })]
+        [System.Int32]
+        $PinMinimumLength,
+
+        [Parameter()]
+        [ValidateScript({ ($_ -ge 4) -and ($_ -le 127) })]
```

Comment:
I think a ValidateRange is better suitable here:
https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/validaterange-attribute-declaration